_posts/2017-11-21-javascript-modules.md

```diff
@@ -0,0 +1,216 @@
+---
+layout: post
+title: JavaScript Modules
+date: '2017-11-21'
+---
+
+Today, I looked a bit closer at JavaScript Modules. I have been using modules since I started using JavaScript but I must admit I have never looked into the subject deeply. Using [Babel](https://babeljs.io/) and [webpack](https://webpack.js.org/) things have been working fine and I just assumed webpack built a sort of dependency tree and Babel helped it understanding the latest ES module syntax (import/export).
```

Comment:
Maybe consider adding a brief note summarising what webpack and babel are? You kind of have it but it's not super clear to me, although i'm also quite new to JS stuff so might just be me (:

Reply:
ok, I added very brief descriptions.